logp/core.go

```diff
@@ -246,13 +246,13 @@ func makeFileOutput(cfg Config) (zapcore.Core, error) {
 		return nil, errors.Wrap(err, "failed to create file rotator")
 	}
 
-	return newCore(cfg, buildEncoder(cfg), rotator, cfg.Level.ZapLevel()), nil
+	return newCore(buildEncoder(cfg), rotator, cfg.Level.ZapLevel()), nil
 }
 
-func newCore(cfg Config, enc zapcore.Encoder, ws zapcore.WriteSyncer, enab zapcore.LevelEnabler) zapcore.Core {
-	return wrappedCore(cfg, zapcore.NewCore(enc, ws, enab))
+func newCore(enc zapcore.Encoder, ws zapcore.WriteSyncer, enab zapcore.LevelEnabler) zapcore.Core {
+	return wrappedCore(zapcore.NewCore(enc, ws, enab))
 }
-func wrappedCore(cfg Config, core zapcore.Core) zapcore.Core {
```

Comment:
the parameter was unused.

Reply:
This will impact beats/fleet I think

Reply:
Is `elastic-agent-libs` intended to be a drop-in replacement for beats/libbeat?

Reply:
@ph how come? this is a private function and the entire package is in this repo now, is not it?

Reply:
I've missed the private function part.